lib/measures/export_modelica_loads/measure.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <measure>
-  <schema_version>3.1</schema_version>
+  <schema_version>3.0</schema_version>
```

Comment:
I dunno how this got changed to something OS doesn't play nicely with, but I'm glad it got standardized! ❤️